features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/Create.jsx

```diff
@@ -70,7 +70,7 @@ const styles = theme => ({
     },
     button: {
         marginLeft: theme.spacing(2),
-        color: theme.palette.getContrastText(theme.palette.primary.main),
+        color: theme.palette.getContrastText(theme.palette.background.paper),
```

Comment:
Used Contrast of background for Paper here because the button is a text variant and on top of Paper component.
Primary Contrast can show up as white on white.